test/framework/src/main/java/org/elasticsearch/test/InternalTestCluster.java

```diff
@@ -337,7 +337,7 @@ public InternalTestCluster(long clusterSeed, Path baseDir,
         builder.put(DiskThresholdSettings.CLUSTER_ROUTING_ALLOCATION_HIGH_DISK_WATERMARK_SETTING.getKey(), "1b");
         builder.put(DiskThresholdSettings.CLUSTER_ROUTING_ALLOCATION_DISK_FLOOD_STAGE_WATERMARK_SETTING.getKey(), "1b");
         // Some tests make use of scripting quite a bit, so increase the limit for integration tests
-        builder.put(ScriptService.SCRIPT_MAX_COMPILATIONS_PER_MINUTE.getKey(), 1000);
+        builder.put(ScriptService.SCRIPT_MAX_COMPILATIONS_RATE.getKey(), "1000/1m");
```

Comment:
As a follow up, I wonder if we really need to support this enormous rate, or if the tweaks to the default might now be sufficient for test clusters?

Reply:
yeah, trying to stick with the new default by default might make most sense... I'll give it a go tomorrow along with some test runs.